### PR TITLE
Releasing 2.1

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,52 @@
+snapcraft (2.0.1ubuntu1) xenial; urgency=medium
```
Comment (kyrofa) [inline debian/changelog]: Not 2.1?

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,52 @@
...
+  * Add document about how to specify app metadata (commands, daemons,
+    config). (LP: #1535905)
+  * Add document about debugging snaps. (LP: #1538016)
+  * Try to capture what the articles are about in a more concise and
+    clearer way. (LP: #1539502)
+  * Add more links between the individual articles. (LP: #1539540)
+
+  [ Reto Breitenmoser ]
+  * tar-content plugin: Add capability to unpack into a specific
+    destination directory LP: #1478055
```
Comment (kyrofa) [inline debian/changelog]: Consider surrounding the bug reference with parentheses for consistency.

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,52 @@
...
+  * Upload: Clean up output. (LP: #1538692, #1539814)
+
+  [ Alexander Sack ]
+  * fix FD leakage by closing apt cache in repo.py (LP: #1537705)
+
+  [ Ricardo Kirkner ]
+  * added support for uploading snap packages directly to the Ubuntu
+    Store. LP: #1538657
+  * make sure otp is included in login request only if it's not
+    empty LP: #1539208
```
Comment (kyrofa) [inline debian/changelog]: Should we capitalize these to make them all pretty-like?

File: debian/changelog (hunk 1 above)
Comment (sergiusens) [inline debian/changelog]: argh, I thought about it but did not change it :-/

File: debian/changelog (hunk 3 above)
Comment (sergiusens) [inline debian/changelog]: yes

File: debian/changelog (hunk 2 above)
Comment (sergiusens) [inline debian/changelog]: I wonder how this wasn't automatic

**Comment by sergiusens** [comment]
@kyrofa thanks and done

File: debian/changelog [hunk 4]
```diff
@@ -1,3 +1,52 @@
+snapcraft (2.1) xenial; urgency=medium
+
+  [ Leo Arias ]
+  * Run the tests in a xenial LXC. LP: #1532213
```
Comment (kyrofa) [inline debian/changelog]: One more here. Why is this happening?

File: debian/changelog [hunk 5]
```diff
@@ -1,3 +1,52 @@
...
+  * Upload: Clean up output. (LP: #1538692, #1539814)
+
+  [ Alexander Sack ]
+  * Fix FD leakage by closing apt cache in repo.py (LP: #1537705)
+
+  [ Ricardo Kirkner ]
+  * Added support for uploading snap packages directly to the Ubuntu
+    Store. LP: #1538657
+  * Make sure otp is included in the login request only if it's not
+    empty LP: #1539208
```
Comment (kyrofa) [inline debian/changelog]: Still got these.

Comment (sergiusens) [inline debian/changelog]: darn, I only looked under the one you mentioned thinking it was only one occurrence :-)

File: debian/changelog (hunk 4 above)
Comment (sergiusens) [inline debian/changelog]: I have no idea!

Comment (sergiusens) [inline debian/changelog]: to be fair, `gbp` is not as good as `bzr bd` so that may be an indication.

**Comment by kyrofa** [comment]
There, you have my :+1: now.

File: debian/changelog (hunk 4 above)
Comment (kyrofa) [inline debian/changelog]: Blasphemer!

Comment (elopio) [inline debian/changelog]: ugh, I was so close this time to get all my commit messages perfect. Damn!

**Pull request merged**